spacy/pipeline/textcat.py

```diff
@@ -208,6 +221,9 @@ def set_annotations(self, docs: Iterable[Doc], scores) -> None:
         DOCS: https://spacy.io/api/textcategorizer#set_annotations
```

Comment:
Even though the `textcat` only has one relevant field here, I would make the signature consistent and have `predict` return a dict, and `set_annotations` take that dict instead of the raw `scores`.

Reply:
Fixed.